infrastructure/serve/cosmos.tf

```diff
@@ -0,0 +1,73 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+resource "azurerm_cosmosdb_account" "serve" {
+  name                = "cosmos-db-${var.naming_suffix}"
+  location            = var.core_rg_location
+  resource_group_name = var.core_rg_name
+  offer_type          = "Standard"
+  kind                = "GlobalDocumentDB"
+  # public_network_access_enabled = var.local_mode
+  tags = var.tags
+
+  capabilities {
+    name = "EnableServerless"
```

Comment:
Have you tested this? Docs say if you have server less enabled free tier doesn't apply

Reply:
I have but not with the free tier because of the [potential multiple instances](https://github.com/UCLH-Foundry/FlowEHR/pull/79#discussion_r1109015306).. is Standard+serverless basically free if it's not being used?

Reply:
Serverless would be free if not being used yes, so I would imagine we enable free tier for `app_dev` (see my reply to your other comment) and then serverless for `prod`

Reply:
> we enable free tier for `app_dev`

how do we do that if `app_dev` is being deployed into a sub with multiple instances?

Reply:
Only `app_dev` has free tier enabled in its config?

Reply:
but free tier is 1 per subscription?

Reply:
Yep, so that would be used by the persistent `app_dev` environment, every other deployment would use serverless I guess as the cheapest alternative

Reply:
Let's come back to this later if still needed!